#For publishing movement commands to drone from swarm
#Identifies drone
string drone_id

# int movement command
int8 movement_command
